src/test/java/bio/terra/service/dataset/DatasetControlFilesIntegrationTest.java

```diff
@@ -222,6 +226,30 @@ public void testSourcePathAuth() throws Exception {
         hasSize(6));
   }
 
+  @Test
+  public void testDirectIngestSourcePathAuth() throws Exception {
+    DatasetSummaryModel datasetSummaryModel =
+        dataRepoFixtures.createDataset(steward(), profileId, "dataset-ingest-combined-array.json");
+    UUID datasetId = datasetSummaryModel.getId();
+    Map<String, Object> data =
+        jsonLoader.loadObject("test-direct-ingest-auth.json", new TypeReference<>() {});
+    IngestRequestModel request = dataRepoFixtures.buildSimpleIngest("sample_vcf", List.of(data));
+    DataRepoResponse<IngestResponseModel> ingestResponse =
+        dataRepoFixtures.ingestJsonDataRaw(steward(), datasetId, request);
+
+    assertThat(
+        "The ingest fails if there are any errors accessing the source path(s)",
+        ingestResponse.getErrorObject().isPresent(),
+        is(true));
```

Comment:
Instead of `assertThat(x, is(true))` you can write `assertThat(x)`. Although in this case may be a matcher for this case so you could write `assertThat(ingestResponse.getErrorObject(), <something>)`. (A quick google search turned up https://github.com/npathai/hamcrest-optional)

Reply:
That's a good point! I copied this over from another test so I'll make a note to clean that up so I can get this into the current sprint :)